### ISSUE TITLE
b550m pro p 

### ISSUE THREAD

**Issue by GELTOSH** [PsaTyrE/Opencore-5600x-B550M#1]
can you create b550m pro p efi for ventura because this efi is no more longer to use ventura and i create opencore patcher to create ventura but the efi is not supported help me please.

**Comment by PsaTyrE** [comment]
sorry for replying just now, what cpu and gpu are you using right now?

**Comment by PsaTyrE** [comment]
> can you create b550m pro p efi for ventura because this efi is no more longer to use ventura and i create opencore patcher to create ventura but the efi is not supported help me please.

hi, if you use the Keplek GT 710 gpu then the patch is not supported until venture

**Comment by GELTOSH** [comment]
Yes sir gt720 bigsur and when use other gpu the monitor is pink screen how to fix it?

**Comment by GELTOSH** [comment]
B550m pro p 5600x gt720 but when i use rx580 4gb my monitor is pink screen.

**Comment by PsaTyrE** [comment]
okay you can try efi it works for me with RX 570 4gb please feedback me if you find the same error
[EFI.zip](https://github.com/PsaTyrE/Opencore-5600x-B550M/files/11381382/EFI.zip)

**Comment by PsaTyrE** [comment]
![myxample](https://user-images.githubusercontent.com/113742594/235887975-372bcd11-b9dd-4f0a-ac65-a16f6532c8e4.jpg)
this work for me

**Comment by PsaTyrE** [comment]
> okay you can try efi it works for me with RX 570 4gb please feedback me if you find the same error [EFI.zip](https://github.com/PsaTyrE/Opencore-5600x-B550M/files/11381382/EFI.zip)

if it works, i will update it in git

**Comment by GELTOSH** [comment]
Ok sir ill try again.

**Comment by GELTOSH** [comment]
![image](https://user-images.githubusercontent.com/128715610/235890372-6d0f364c-b084-4909-9df8-3b0dbed9fb89.jpeg) maybe faulty gpu i think

**Comment by GELTOSH** [comment]
Hello can you create bootable design like this on our efi thanks![image](https://user-images.githubusercontent.com/128715610/235891674-ef28b5b4-3385-4e88-8ac1-7e349a71989e.jpeg)

**Comment by PsaTyrE** [comment]
sorry lol, I can't do that, but you might be able to grab it here
https://dortania.github.io/OpenCanopy-Gallery/blackosx.html#themes

**Comment by PsaTyrE** [comment]
> ![image](https://user-images.githubusercontent.com/128715610/235890372-6d0f364c-b084-4909-9df8-3b0dbed9fb89.jpeg) maybe fault gpu i think

it looks like you have a problem with your gpu or the HDMI/DP cable

**Comment by GELTOSH** [comment]
Hello this efi is supported in ventura?

**Comment by PsaTyrE** [comment]
hello, not support, you need to update the patch before up to venture

**Comment by GELTOSH** [comment]
Thanks
